services/gateway_manager/src/main/resources/application.properties

```diff
@@ -0,0 +1,22 @@
+server.port=9015
+
```

Comment:
@xieus Does this port for GM? please confirm.

Reply:
I am fine with 9015 and K8s cluster port of 30015. Keep in mind that 9014 remain available therefore we can keep it for other microservice.